lib/enumerize/attribute.rb

```diff
@@ -17,6 +17,13 @@ def initialize(klass, name, options={})
         @default_value = find_default_value(options[:default])
         raise ArgumentError, 'invalid default value' unless @default_value
       end
+
+      # Define methods to get each value
+      @values.each do |v|
+        metaclass = class << self; self; end
+        metaclass.send(:define_method, v.to_s) { v.value }
+        metaclass.send(:define_method, v.to_s + '_text' ) { v.text }
+      end
```

Comment:
These methods aren't getting defined on the singleton classes of the `ActiveRecord` instance, they are only defined on the `Enumerize::Attribute` instance.  This wouldn't affect the serialization of the `ActiveRecord` instance (in our example, instances of the `Client`) class, it would only be an issue if they tries to serialize the internal `Enumerize::Attribute` object.  People shouldn't be doing that anyways, so I don't think that it is an issue.  

`method_missing` hacks are more expensive (and complicated) than a method defined on the singleton class, so I recommend the method definitions for performance reasons.

What do you think?